app/models/user.rb

```diff
@@ -13,6 +13,14 @@ class User < ApplicationRecord
     password_salt&.last(10)
   end
 
+  def isAdmin
+    role.downcase === "admin"
+  end
+
+  def isMember
+    role.downcase === "member"
+  end
```

Comment:
These are not necessary since we've got an enum.  You should be able to already call `User.first.member?` and `User.first.admin?`.

Reply:
Gotcha